modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/UploadTempImageMVCActionCommand.java

```diff
@@ -41,15 +51,37 @@ protected void doProcessAction(
 			_itemSelectorUploadResponseHandler, actionRequest, actionResponse);
 	}
 
+	@Reference
+	private DLValidator _dlValidator;
+
 	@Reference
 	private ItemSelectorUploadResponseHandler
 		_itemSelectorUploadResponseHandler;
 
+	@Reference
+	private MBMessageService _mbMessageService;
+
 	@Reference
 	private TempImageMBUploadFileEntryHandler
```

Comment:
Hey @regiCesar21, you forgot to remove the @Reference here. You did the same in the Edit* one.